styles/mixins/_focus.scss

```diff
@@ -39,9 +39,10 @@ $-color-focus-outline: var.$color-focus-outline;
  */
 @mixin outline-on-keyboard-focus($inset: false) {
   @include outline($inset);
+
   // Selector for browsers using JS polyfill, which adds the "focus-visible"
```

Comment:
I suggest we replace the phrase "JS polyfill" here with a link to the specific polyfill we assume is being used: https://github.com/WICG/focus-visible. The README for this project has an example which matches what you are doing here.

Reply:
Great suggestion! Comment updated and expanded.